test/models.spec.ts

```diff
@@ -529,12 +529,93 @@ describe("Unit | Filters", function () {
     });
   });
 
+  describe('validateFiltersContainer', function () {
+    it("validator should return errors if object does NOT validate against filtersContainer schema", function () {
+      // Arrange
+      const malformedFiltersContainer = {
+        abc: '123'
+      };
+
+      // Act
+      const errors = models.validateFiltersContainer(malformedFiltersContainer);
+
+      // Assert
+      expect(errors).toBeDefined();
+    });
+
+    it("validator should return no errors if object does validate against filtersContainer schema", function () {
+      // Arrange
+      const things: (string | number)[] = [
+        "a",
+        3234
+      ];
+
+      const basicFilter: models.IBasicFilter = {
+        $schema: "a",
+        target: {
+          table: "table",
+          column: "column"
+        },
+        operator: "In",
+        values: [
+          "A",
+          "B"
+        ]
+      };
+      const advancedFilter: models.IAdvancedFilter = {
+        $schema: "a",
+        target: {
```

Comment:
This is what I asked about at stand up.  I'm not sure what to change it to.  I was thinking level or scope.  The number of visuals affected by the filter is determined by the scope? (Single visual, All visuals on page, all visuals in report)